src/com/oltpbenchmark/CommandLineOptions.java

```diff
@@ -91,6 +92,7 @@ public Mode getMode() {
         if (isBooleanOptionSet("create")) return Mode.CREATE;
         if (isBooleanOptionSet("load")) return Mode.LOAD;
         if (isBooleanOptionSet("execute")) return Mode.EXECUTE;
+        if (isBooleanOptionSet("enable-foreign-keys")) return Mode.ENABLE_FOREIGN_KEYS;
```

Comment:
where do we actually use this?

Reply:
the experiments with multiple clients need this to be a separate set.
Example : https://docs.yugabyte.com/latest/benchmark/tpcc-ysql/#10000-wh

Reply:
ok, but where in this change do we use `ENABLE_FOREIGN_KEYS` and behave differently?

Reply:
ok, but we should be getting the result of this method somewhere and using it. is there somewhere that we're manually looking at "enable-foreign-keys" instead of using this method? can we fix that as part of this change as well?

Reply:
Done.